tests/common/tests/cca_test.hpp

```diff
@@ -107,7 +107,7 @@ class ConnectedComponentAnalysisTests
                 result.at(io_truth.geometry_id);
 
             traccc::scalar tol = std::max(
-                0.01, 0.0001 * std::max(io_truth.channel0, io_truth.channel1));
+                0.1, 0.0001 * std::max(io_truth.channel0, io_truth.channel1));
```

Comment:
😕  Did the tolerance worsen as a result of all of this? I thought if anything, this would make the test more accurate...

Reply:
Well. the test didn't pass the variance test after changes.
```
            EXPECT_NEAR(match->variance[0], io_truth.variance0, tol);
            EXPECT_NEAR(match->variance[1], io_truth.variance1, tol);
```
TBH I don't know how this test works and data was created :/ But it's not surprising at all as out variance estimation is not that robust

Reply:
I think I figured out the reason.
The default pitch size of  `pixel_data` is 1 which makes the reconstructed variance (`match->variance[0]` and `match->variance[1]` ) 1/12 or ~0.08.
And the `io_truth.variance0` (which is truth input from synthetic data) is 0.
That's why `0.01` tolerance doesn't work here.

Then someone will ask why it has been OK so far.
Before this PR, `pixel_data` of `cell_module` gets the default pitch size of 0.05:
```
pixel_data pixel{-8.425, -36.025, 0.05, 0.05};
```
This makes the reconstructed variance `0.05/12`, which is fine with 0.01 tolerance. 
This PR removes the default value of `pixel_data` and causes the unit test fails with 0.08 variance.

The conclusion is `cca_test` or synthetic data needs some modification :p...